test/testdata/deployednettemplates/recipes/scenario1/net.json

```diff
@@ -172,6 +172,7 @@
 							"ParticipationOnly": false
 						}
 					],
+					"APIEndpoint": "{{APIEndpoint}}",
```

Comment:
this file is auto-generated. It's ok to merge in the changes here, but you need to track down the source. ( i.e. I think it's `node.json` and `nonPartNode.json`

Reply:
bump

Reply:
Had to add a little code to the generator, but this is covered now.

Reply:
I'm not sure if the code generator changes were needed. Could you verify that specifying APIEndpoint/APIEndpoint2/APIEndpoint3/APIEndpoint4 wouldn't have worked ?
( btw - even if you "mess-up", the worst case scenario would be a port collision, in which case one of the processes would still be working at the desired port.. right ? )

Reply:
The generator would have needed to change to apply APIEndpoint{N} to different nodes within a host. APIEndpoint[234] appears to only exist on a couple hand written templates. I would expect `algod` starting up and failing to open its APIEndpoint at the specified port would exit; then we'd have fewer nodes for all the colliding nodes.